.github/workflows/package-mac.yml

```diff
@@ -0,0 +1,28 @@
+name: Package GEDKeeper3 for MacOS
+
+on:
+  push:
+    branches: [ "master" ]
+  pull_request:
+    branches: [ "master" ]
+
+jobs:
+  mac:
+
+    runs-on: macos-12
+
+    steps:
+    - uses: actions/checkout@v3
+    - name: Setup .NET
+      uses: actions/setup-dotnet@v3
+      with:
+        dotnet-version: 6.0.x
+    - name: Build
+      run: dotnet build -c MacOS_Debug -p:EnableDmgBuild=true
+      working-directory: projects/GKv3
+    - name: Upload DMG file
```

Comment:
It crates an artifact that can be downloaded from Actions tab (Actions -> 'Package GEDKeeper 3 for MacOS' -> workflow run, at the bottom of 'Summary' page). Then it can be downloaded from there and distributed as a normal release file.